envs/conda_build_config.yaml

```diff
@@ -90,7 +90,7 @@ googleapis_common_protos:
 gmock:
   - 1.10.0
 grpc_cpp:
-  - 1.36.*
+  - 1.41.*
```

Comment:
Am assuming that the feedstock would build the conda package fine on s390x also. Please let me know if it fails on s390x.